tasks/InsightAppSec/helpers/insightAppSecApi.ts

```diff
@@ -1,13 +1,28 @@
+const AXIOS = require('axios').default;
+const LOCATION_HEADER = "location";
+const CONTENT_TYPE_HEADER = "application/json";
+const ACCEPT_HEADER = "application/json";
+const USER_AGENT_HEADER = "r7:insightappsec-azure-devops-extension/1.0.7";
 
 export default class InsightAppSecApi
 {
     endpoint: string;
     apiKey: string;
+    axiosInst;
 
     constructor(endpoint, apiKey)
     {
         this.endpoint = endpoint;
-        this.apiKey = apiKey;
+        this.axiosInst = AXIOS.create({
+            baseURL: endpoint,
+            headers: {"X-Api-Key": apiKey,
+                      "Content-Type": CONTENT_TYPE_HEADER,
+                      "Accept": ACCEPT_HEADER,
+                      "User-Agent": USER_AGENT_HEADER},
```

Comment:
Another minor nit pick, looks like the formatting needs fixed:
headers: {
  "Content-Type":...
  ...
},